BuildDetails.md

```diff
@@ -0,0 +1,55 @@
+
+### Table of Contents
+
+1. [How to Build](#how-to-build)
+2. [Grunt Tasks](#grunt-tasks)
+
+## How to Build
+In order to build a FullScreenShenanigans project you must use Grunt. To do so, open the terminal and navigate to the project directory. You can build the project all at once or step-by-step. If you wish to build it in one go, enter `grunt`. This command will run all of the tasks specified in the [Grunt Tasks](#grunt-tasks) section of this document. Alternatively, you can run each grunt task manually. This allows you to run part of the build at a time. The different tasks are outlined in the [Grunt Tasks](#grunt-tasks) section of this document.
+
+
+## Grunt Tasks
+The following ten tasks are run, in order, every time you build the project with grunt. The end result of the build is a new directory named Distribution. This directory will contain all of the project files in minified form.
+
+### clean
+The clean task deletes all files and folders in the project's Distribution directory. This occurs because the Distribution folder must be recreated to reflect any changes made to the files. See [this](https://github.com/gruntjs/grunt-contrib-clean) page for more information on the task.
+
+### tslint
+In order to make sure that the .ts files in the project are neat and readable, grunt runs tslint. This task checks each file for whitespaces, alignment issues, unused variables, missing semicolons, and more, throwing errors if it finds any issues. See [this](https://github.com/palantir/grunt-tslint) page for more information on tslint.
+
+### typescript
+#### typescript:base
+This task compules each .ts file in the Source directory into a .js file in the same directory.
+#### typescript:distribution
+The typescript:distribution task translates the project's main .ts file into a .js file that is placed in the Distribution directory and concatenated with the version number of the project.
+
+See [this](https://github.com/k-maru/grunt-typescript) page for more information on typescript.
+
+### copy
+This task copies files and folders from the Source directory into the Distribution directory. 
+* It copies the project's main .ts file into Distribution, concatenating the project version to the name, and does the same to the accompanying .d.ts file. 
+* It also copies the file created by typescript:distribution to the directory in Distribution named after the project name and version. 
+* The References directory in Source is copied to the Distribution directory, too. 
+* The README.md and LICENSE.txt files are both copied into Source and Distribution. 
+* The last things copied into the Distribution directory are the Source/Fonts, Source/Sounds, and Source/Theme directories.
```

Comment:
You don't need to say in every bullet that things are copied. You can say it in or after the first sentence.